protocol/testing/testnet-local/local.sh

```
@@ -157,14 +160,24 @@
+
 setup_cosmovisor() {
 	for i in "${!MONIKERS[@]}"; do
 		VAL_HOME_DIR="$HOME/chain/.${MONIKERS[$i]}"
 		export DAEMON_NAME=dydxprotocold
 		export DAEMON_HOME="$HOME/chain/.${MONIKERS[$i]}"
 
-		cosmovisor init /bin/dydxprotocold
-	done
+		cosmovisor init $PREUPGRADE_BINARY_PATH
+		mkdir -p "$VAL_HOME_DIR/cosmovisor/upgrades/v9.0/bin/"
+		ln -s /bin/dydxprotocold "$VAL_HOME_DIR/cosmovisor/upgrades/v9.0/bin/dydxprotocold"	done
```

Shellcheck (0.10.0)

[warning] Use semicolon or linefeed before 'done' (or quote to make it literal).

(SC1010)